test/DynamoCoreTests/UndoRedoRecorderTests.cs

```diff
@@ -609,9 +609,9 @@ public void TestBasicAttributes()
             sumNode.X = 250;
```

Comment:
These properties cannot be set directly now, so go through backdoor API `ModelBase.UpdateValue` method.